styleguide/components/11-base/base/base.js

```diff
@@ -9,7 +9,6 @@ window.gent_styleguide = (function () { // eslint-disable-line no-unused-vars
    * @constructor
    */
   function TabTrap(container) {
-    var focusPosition = -1;
     var focusables = getFocusables(container);
```

Comment:
horen deze vars geen const te zijn?

Reply:
This file predates our shift to ecmascript.